src/components/OptionsSelector.js

```diff
@@ -217,7 +217,7 @@ class OptionsSelector extends Component {
         return (
             <View style={[styles.flex1]}>
                 <View style={[styles.ph5, styles.pv3]}>
-                    <TextInput
+                    <PhoneTextInput
```

Comment:
Why do we need this change?

Reply:
We aren't removing the hyphens and brackets from the search term, using PhoneTextInput to achieve that.

Reply:
We dot have to remove those here.

Reply:
Asked [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1649501031647289?thread_ts=1649454970.498459&cid=C01GTK53T8Q)